src/bash_functions.sh

```diff
@@ -71,6 +60,41 @@ ensure_basic_configuration() {
     fi
 }
 
+setup_web_password() {
+    echo "  [i] Checking web password"
+    # If the web password variable is not set...
+    if [ -z "${FTLCONF_webserver_api_password+x}" ]; then
+        # is the variable FTLCONF_ENV_ONLY set to true?
+        if [ "${FTLCONF_ENV_ONLY}" == "true" ]; then
+            echo "  [i] No password supplied via FTLCONF_webserver_api_password, but FTLCONF_ENV_ONLY is set to true, using default (none)"
+            # If so, return - the password will be set to FTL's default (no password)
+            return
+        fi
+
+        # Exit if password is already set in config file
+        if [[ -n $(pihole-FTL --config webserver.api.pwhash) ]]; then
```

Comment:
Still think this should be something like

```suggestion
        if [[  $(pihole-FTL --config webserver.api.pwhash) = \$BALLOON-SHA256* ]]; then
```

Reply:
Is there an advantage to checking for a value that begins `$BALLOON-SHA256`, vs checking for an empty value?

Reply:
It's more precis - we want to know the command returns a password hash not anything else (e.g. some kind of API error)

Reply:
Cool, fixed in latest commit